attestation-service/verifier/src/lib.rs

```diff
@@ -152,6 +166,12 @@ pub trait Verifier {
         expected_report_data: &ReportData,
         expected_init_data_hash: &InitDataHash,
     ) -> Result<TeeEvidenceParsedClaim>;
+
+    async fn generate_challenge_extra_params(
+        &self,
```

Comment:
I think there may be some extra input parameters when call this function in the future, so can you remain a optional input parameter here? Such as `tee_parameters: Option<Vec<u8>>`.

Reply:
fixed.